src/collections/backup/client.ts

```diff
@@ -261,7 +265,8 @@ export interface Backup {
   /** List existing backups (completed and in-progress) created in a given backend.
    *
    * @param {Backend} backend Backend whence to list backups.
+   * @param {sortAscending} [boolean] Sort list of backups in ascending order based on creation time. Default is descending order.
    * @returns {Promise<BackupReturn[]>} The response from Weaviate.
    * */
-  list(backend: Backend): Promise<BackupReturn[]>;
+  list(backend: Backend, sortAscending?: boolean): Promise<BackupReturn[]>;
```

Comment:
```suggestion
  list(backend: Backend, opts?: ListBackupOptions): Promise<BackupReturn[]>;
```

where ListBackupOptions is defined as follows:

```ts
type ListBackupOptions = { startedAtAsc?: boolean };
```


Two things here:

1. I suggest mentioning the field backups are sorted by in the variable name, i.e. "started at", similarly to what's been done in Go client
2. A pattern for any optional parameters we use across the repo is to declare an -Options type and make it the last optional argument to the function.